recipes/xcube/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set version = "0.2.0" %}
+
+package:
+  name: xcube
+  version: {{ version }}
+
+source:
+  url: https://github.com/dcs4cop/xcube/archive/v{{ version }}.tar.gz
+  sha256: 35a934429ff173b383203395ad4ad513c405a15bd5a6c2a319d4d8fa6816f116
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - xcube = xcube.cli.cli:main
+    - xcube-gen = xcube.cli.gen:main
+    - xcube-restime = xcube.cli.restime:main
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - affine >=2.2
+    - click >=7.0
+    - cmocean >=2.0
+    - dask >=1.2
+    - fiona >=1.8
+    - gdal >=2.4
```

Comment:
Unless you depend on `gdal` directly let `fiona` pull the right version. Your pin, BTW, is wrong b/c conda may try to optimize for gdal 3, latest, but fiona does not work with that version.

Reply:
Removed gdal for the next push